joboffers/tests/test_model.py

```diff
@@ -121,3 +124,76 @@ def test_assert_slug_is_updated_on_title_change():
     joboffer.save()
 
     assert slugify(UPDATED_TITLE) == joboffer.slug
+
+
+@pytest.mark.django_db
+def test_assert_short_description_is_set_with_stripped_description():
+    """
+    Assert that a joboffer short description is created with the splitted description
+    if there is no short description given.
+    """
+    description = SHORT_JOBOFFER_DESCRIPTION
+    short_description = STRIPPED_JOBOFFER_SHORT_DESCRIPTION
+
+    joboffer = JobOfferFactory.create(
+        remoteness=Remoteness.REMOTE,
+        title='Job Offer',
+        location=None,
+        contact_mail=Faker('email'),
+        contact_phone=None,
+        contact_url=None,
+        description=description,
+        short_description='',
+    )
+
+    assert short_description == joboffer.short_description
+
+
+@pytest.mark.django_db
+def test_assert_short_description_is_set_with_the_given_short_description():
+    """
+    Assert that a joboffer is created with the short description given.
```

Comment:
Para que se entienda mejor el objetivo podrías poner algo como:

`Assert that the joboffer doesn't update the short_description if it is provided in the model`